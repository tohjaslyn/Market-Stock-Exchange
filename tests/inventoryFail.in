add trader1 100
inventory trader1
inventory trader2
exit
