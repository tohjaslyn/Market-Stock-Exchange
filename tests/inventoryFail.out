$ Success.
$ Trader has an empty inventory.
$ No such trader in the market.
$ Have a nice day.
